tests/test_basic.py

```diff
@@ -3,16 +3,32 @@
 import json
 import unittest
 
+import redis
+
 from rq_dashboard.cli import make_flask_app
 
+from rq import Worker
+from rq import pop_connection, push_connection
```

Comment:
Please apply isort to It.

Reply:
Good utility! I did not know about this lib until now. Thanks.